src/palace/manager/api/bibliotheca.py

```diff
@@ -768,7 +762,7 @@ def value(bibliotheca_key):
 
         # Presume all images from Bibliotheca are JPEG.
         media_type = Representation.JPEG_MEDIA_TYPE
-        cover_url = value("CoverLinkURL").replace("&amp;", "&")
+        cover_url = self.text_of_subtag(tag, "CoverLinkURL").replace("&amp;", "&")
```

Comment:
This is the same as `value`, but it raises if the element doesn't exist. The element must exist, since we were calling `replace` on it. Otherwise we would be calling `replace` on `None`. So the logic here is equivalent in the normal case, and gives a more descriptive error in the error case.